http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -194,6 +194,30 @@ class SamAuthProvider[F[_]: OpenTelemetryMetrics](
     } yield roles.contains(SamRole.Owner)
   }
 
+  def filterWorkspaceReader(
+    resources: NonEmptyList[WorkspaceResourceSamResourceId],
```

Comment:
It seems like `resources` is not used in this method.. it doesn't seem to be filtering anything, just returning all the workspace resource IDs from Sam. I'll further to try and understand better how this method is used..

Reply:
Oh yeah, I need to clean this up, probably too hastily copied it from the `filterWorkspaceOwner` function 😬

Reply:
Yeah I see `filterWorkspaceOwner` behaves the same way.. and I think the filtering _is_ occurring, just outside of this method (see below comment).